src/main/scala/cromwell/engine/db/slick/DataAccessController.scala

```diff
@@ -0,0 +1,310 @@
+package cromwell.engine.db.slick
+
+import java.util.{UUID, Date}
+
+import cromwell.binding.Call
+import cromwell.binding.types.WdlType
+import cromwell.binding.values.{WdlPrimitive, WdlValue}
+import cromwell.engine.db._
+import cromwell.engine.store.ExecutionStore.ExecutionStatus
+import cromwell.engine.store.SymbolStore.SymbolStoreKey
+import cromwell.engine.store.SymbolStoreEntry
+import cromwell.engine.{WorkflowId, WorkflowState, WorkflowSubmitted}
+import cromwell.util.DatabaseConfig
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ExecutionContext, Await, Future}
+import scala.language.implicitConversions
+
+// TODO: Need lots of refactoring including: better futures handling, compiled queries, utility functions, etc.
+object DataAccessController extends DataAccess {
+  val dataAccess = new DataAccessComponent(DatabaseConfig.slickDriver)
+
+  import dataAccess.driver.api._
+
+  def database: Database = Database.forConfig("", DatabaseConfig.databaseConfig)
+
+  // TODO: Pass in the execution context instead of using the global!!
+  private implicit val executionContext = ExecutionContext.global
+
+  private implicit def date2timestamp(date: java.util.Date): java.sql.Timestamp =
```

Comment:
yeah I think I agree... it seems to me that the way it's being used now, it doesn't need to be implicit.  Personally, since this is really only being used in this file, why not just take the `implicit` off?